assets/js/admin/wc-setup.js

```diff
@@ -1,99 +1,95 @@
-				} else {
-					$( '.tax-rates' ).hide();
-				}
-			}
-		} else {
-			$(':input[name="currency_pos"]').closest('tr').show();
-			$(':input[name="thousand_sep"]').closest('tr').show();
-			$(':input[name="decimal_sep"]').closest('tr').show();
-			$(':input[name="num_decimals"]').closest('tr').show();
-			$( '.tax-rates' ).hide();
-		}
-	} ).change();
-
-	$( 'input[name="woocommerce_calc_taxes"]' ).change( function() {
-		if ( $(this).is( ':checked' ) ) {
-			$(':input[name="woocommerce_prices_include_tax"], :input[name="woocommerce_import_tax_rates"]').closest('tr').show();
-			$('tr.tax-rates').show();
-		} else {
-			$(':input[name="woocommerce_prices_include_tax"], :input[name="woocommerce_import_tax_rates"]').closest('tr').hide();
-			$('tr.tax-rates').hide();
-		}
-	} ).change();
-
-	$( '.button-next' ).on( 'click', function() {
+	function blockWizardUI() {
 		$('.wc-setup-content').block({
 			message: null,
 			overlayCSS: {
 				background: '#fff',
 				opacity: 0.6
 			}
 		});
+	}
+
+	$( '.button-next' ).on( 'click', function() {
+		var form = $( this ).parents( 'form' ).get( 0 );
+
+		if ( ( 'function' !== typeof form.checkValidity ) || form.checkValidity() ) {
+			blockWizardUI();
+		}
+
 		return true;
 	} );
 
-	$( '.wc-wizard-payment-gateways, .wc-wizard-shipping-methods' ).on( 'change', '.wc-wizard-gateway-enable input, .wc-wizard-shipping-enable input', function() {
+	$( '.wc-wizard-services' ).on( 'change', '.wc-wizard-service-enable input', function() {
 		if ( $( this ).is( ':checked' ) ) {
-			$( this ).closest( 'li' ).addClass( 'checked' );
+			$( this ).closest( '.wc-wizard-service-toggle' ).removeClass( 'disabled' );
 		} else {
-			$( this ).closest( 'li' ).removeClass( 'checked' );
+			$( this ).closest( '.wc-wizard-service-toggle' ).addClass( 'disabled' );
 		}
 	} );
 
-	$( '.wc-wizard-payment-gateways, .wc-wizard-shipping-methods' ).on( 'click', 'li.wc-wizard-gateway, li.wc-wizard-shipping', function() {
-		var $enabled = $( this ).find( '.wc-wizard-gateway-enable input, .wc-wizard-shipping-enable input' );
+	$( '.wc-wizard-services' ).on( 'click', '.wc-wizard-service-enable', function( e ) {
+		e.stopPropagation();
+
+		var $checkbox = $( this ).find( '.wc-wizard-service-toggle input' );
+		$checkbox.prop( 'checked', ! $checkbox.prop( 'checked' ) ).change();
+	} );
 
-		$enabled.prop( 'checked', ! $enabled.prop( 'checked' ) ).change();
+	$( '.wc-wizard-services-list-toggle' ).on( 'change', '.wc-wizard-service-enable input', function() {
+			$( this ).closest( '.wc-wizard-services' ).find( '.wc-wizard-service-item' )
+				.slideToggle()
+				.css( 'display', 'flex' );
 	} );
 
-	$( '.wc-wizard-payment-gateways li, .wc-wizard-shipping-methods li' ).on( 'click', 'table, a', function( e ) {
-		e.stopPropagation();
+	$( '.wc-wizard-services' ).on( 'change', '.wc-wizard-shipping-method-select .method', function( e ) {
+		var $zone = $( this ).parent( 'div' );
+		var selected_method = e.target.value;
```

Comment:
What's the coding style? `snake_case` for variables, but `camelCase` for functions?